uidefinitions/createUiDefinition.json

```diff
@@ -229,7 +229,7 @@
                 "constraints": {
                   "allowedValues": [
                     {
-                      "label": "NFS",
+                      "label": "[if(equals(steps('filesystem').shared.newexisting,'new'),'NFS on Scheduler VM','')]",
```

Comment:
I would prefer the text to match the CycleCloud cluster selection, so "Builtin NFS" rather than "NFS on Scheduler VM"